cli/cmd/send/send.go

```diff
@@ -37,6 +41,7 @@ func Run(cmd *cobra.Command, _ []string) {
 
 	urls, _ := cmd.Flags().GetStringSlice("url")
 	message, _ := cmd.Flags().GetString("message")
+	title, _ := cmd.Flags().GetString("title")
```

Comment:
The cyclomatic complexity of `Run` is >10. I suggest refactoring and extracting from it to make it easier to reason about (and test)

Reply:
Yeah, I agree, but skipping it in this branch. The plan is to rewrite this command as a whole pretty soon.